virtcontainers/container.go

```diff
@@ -623,7 +623,10 @@ func newContainer(sandbox *Sandbox, contConfig ContainerConfig) (*Container, err
 // been performed before the container creation failed.
 // - Unplug CPU and memory resources from the VM.
 // - Unplug devices from the VM.
-func (c *Container) rollbackFailingContainerCreation() {
+func (c *Container) rollbackFailingContainerCreation(s *Sandbox) {
```

Comment:
You can always find `sandbox` in `c.sandbox`, so there is no need to pass in another sandbox as argument here.

Reply:
code updated, thanks @bergwolf ~